cmd/edit/cluster/cmd.go

```diff
@@ -176,7 +231,9 @@ func run(cmd *cobra.Command, _ []string) {
 		changedFlags := false
 		for _, flag := range []string{"expiration-time", "expiration", "private",
 			"disable-workload-monitoring", "http-proxy", "https-proxy", "no-proxy",
-			"additional-trust-bundle-file", "additional-allowed-principals", "audit-log-arn"} {
+			"additional-trust-bundle-file", "additional-allowed-principals", "audit-log-arn",
+			"allowed-registries", "blocked-registries", "insecure-registries", "allowed-registries-for-import",
+			"platform-allowlist", "registry-additional-trusted-ca"} {
 			if cmd.Flags().Changed(flag) {
 				changedFlags = true
 			}
```

Comment:
As soon as we set `changeFlags` to `true` there is no reason to keep iterating
```suggestion
			if cmd.Flags().Changed(flag) {
				changedFlags = true
				break
			}
```